### PR TITLE
Fix autowire issue

### PR DESCRIPTION
**Pull request by JaisDK**

### CODE REVIEW

**Comment by JaisDK** [comment]
Fix for #5

File: README.md [hunk 1]
```diff
@@ -24,32 +24,76 @@ $ composer require setono/sylius-pickup-point-plugin
...
-### Step 3: Import routing
+### Step 2: Import routing
 
 ````yaml
 setono_sylius_pickup_point_plugin:
     resource: "@SetonoSyliusPickupPointPlugin/Resources/config/routing.yml"
 ````
 
-### Step 4: Addition of a validation group `checkout_select_shipping`
+### Step 3: Addition of a validation group `checkout_select_shipping`
```
Comment (loevgaard) [inline README.md]: Why do we need this?

File: README.md [hunk 2]
```diff
@@ -24,32 +24,76 @@ $ composer require setono/sylius-pickup-point-plugin
...
+````xml
+<?xml version="1.0" encoding="UTF-8"?>
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                                      http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <mapped-superclass name="App\Entity\Shipment" table="sylius_shipment">
+        <field name="pickupPointId" column="pickup_point_id" length="191" nullable="true" />
```
Comment (loevgaard) [inline README.md]: Leave out the `length` attribute

File: README.md [hunk 3]
```diff
@@ -24,32 +24,76 @@ $ composer require setono/sylius-pickup-point-plugin
...
+````xml
+<?xml version="1.0" encoding="UTF-8"?>
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                                      http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <mapped-superclass name="App\Entity\ShippingMethod" table="sylius_shipping_method">
+        <field name="pickupPointProvider" column="pickup_point_provider" length="191" nullable="true" />
```
Comment (loevgaard) [inline README.md]: Leave out the `length` attribute

File: tests/Application/src/AppBundle/Resources/config/doctrine/model/Shipment.orm.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
                                       http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
     <mapped-superclass name="AppBundle\Model\Shipment" table="sylius_shipment">
-        <field name="pickupPointId" column="pickup_point_id" type="string" length="191" nullable="true" />
+        <field name="pickupPointId" column="pickup_point_id" length="191" nullable="true" />
```
Comment (loevgaard) [inline tests/Application/src/AppBundle/Resources/config/doctrine/model/Shipment.orm.xml]: Leave out the `length` attribute

File: tests/Application/src/AppBundle/Resources/config/doctrine/model/ShippingMethod.orm.xml [hunk 5]
```diff
@@ -6,7 +6,7 @@
                                       http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
     <mapped-superclass name="AppBundle\Model\ShippingMethod" table="sylius_shipping_method">
-        <field name="pickupPointProvider" column="pickup_point_provider" type="string" length="191" nullable="true" />
+        <field name="pickupPointProvider" column="pickup_point_provider" length="191" nullable="true" />
```
Comment (loevgaard) [inline tests/Application/src/AppBundle/Resources/config/doctrine/model/ShippingMethod.orm.xml]: Leave out the `length` attribute

File: README.md (hunk 1 above)
Comment (JaisDK) [reply README.md]: I'm guessing it's to register the validator src/Resources/config/validation/Shipment.xml but I didn't make it so I don't know if it's the correct way.

Comment (JaisDK) [reply README.md]: It's required

**Pull request merged**